inc/api-extensions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+function hale_extend_wp_api($data, $post, $context) {
+
+    // Add featured image to API
+    $featured_image_id = $data->data['featured_media']; // get featured image id
```

Comment:
What happens if featured image is off? No array key?

Reply:
Added `array_key_exists` to check for that.